vignettes/generic.Rmd

```diff
@@ -79,7 +79,7 @@ ae <- filter(ae, USUBJID %in% c("01-701-1111", "01-701-1047")) %>%
   ))
 ```
 
```

Comment:
I'm thinking name of the vignette should be Merged, Joined and Extreme Functions.  Generic still doesn't sit well with me.

Reply:
@bms63 i would leave this otherwise we'll be updating it every time we invent a new generic function. Generic works for me as they're all functions that serve multi-purposes.